data/proposals.js

```diff
@@ -1,57 +1,146 @@
 export default [
+   {
+    id: 999999,
```

Comment:
Can you fill this one out with the EIP999? Or just add the EIP999 and make visible to false

Reply:
It is set visible: False just below

Reply:
This is a template right? We still want the EIP999.. I don't know why you deleted it..

Reply:
I edited the json that already existed and I didn't add back in EIP999 afterwards. I didn't think of it because this site isn't for EIP999. I pulled the first few of each category so we get a good sampling of filtering. It will get added back in with the rest of them at some point.

Reply:
After a few copy, pasting, and overwriting. I realized I wanted to make sure I wasn't leaving in the data from the copied EIP. So to keep transcription errors low I created a template one and gave it a unreasonably large number. so it would stand out if I didn't copy it over right correctly

Reply:
The EIP999 one has the correct author format with Github account name in parenthesis. If you can add that on the other ones that would be fine then